docs/rst/getting_started/tutorial.rst

```diff
@@ -197,99 +200,87 @@ are related with the topics that we are not using will remain without data.
 Create Series Chart
 ===================
 
-In this section we are going to see how to graphically represent the data that is being monitored.
+This section describe how to graphically represent the data that is being monitored.
```

Comment:
This section describes how to graphically represent the data that is being monitored.